files/slurm-resources.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+### BEGIN INIT INFO
+# Provides:          set-resources
+# Required-Start:    $remote_fs $syslog 
+# Required-Stop:     $remote_fs $syslog 
+# X-Start-Before:    slurmctld slurmd
```

Comment:
@ltalirz  this starts it before

Reply:
right - great!

I.e. these options you are adding are needed only if you do the change after startup, correct?

Reply:
anyhow, feel free to proceed without further input from my side!

Reply:
yeh I don't think I can test it any more now lol, spent long enough on it